app/main/views/briefs.py

```diff
@@ -59,16 +59,16 @@ def ask_brief_clarification_question(brief_id):
 
 @main.route('/opportunities/<int:brief_id>/responses/create', methods=['GET'])
 @login_required
-def submit_brief_response(brief_id):
+def brief_response(brief_id):
```

Comment:
Yeah, fair enough. I got the impression we're doing `create_` fro looking at service submissions, but this works too.